buildlogger.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package cedar;
 
-option go_package = "internal";
+option go_package = "rpc/internal";
```

Comment:
This doesn't require a `./` to make it a relative path, but `./gopb` is necessary in juniper?

Reply:
Yeah, it doesn't make sense. This is the error message: 
```
The import path must contain at least one period ('.') or forward slash ('/') character.

See https://developers.google.com/protocol-buffers/docs/reference/go-generated#package for more information.
```